editor/modes/visual-editor/block.js

```diff
@@ -232,7 +232,7 @@ class VisualEditorBlock extends wp.element.Component {
 				tabIndex="0"
 				{ ...wrapperProps }
 			>
-				{ ( showUI || isHovered ) && <BlockMover uids={ [ block.uid ] } /> }
+				{ ( showUI || isHovered ) && <BlockMover uid={ block.uid } order={ this.props.order } /> }
```

Comment:
I think we can't make this change because the block mover is also used with several blocks (multi-selection). See the second usage in this exact same file